tutorials/hello/hello_client/src/app/plugins/hello_info/db.ts

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable consistent-return */
+/* eslint-disable no-console */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+import { codec, db as liskDB, cryptography } from 'lisk-sdk';
+import * as os from 'os';
+import { join } from 'path';
+import { ensureDir } from 'fs-extra';
+import { offChainEventSchema, counterSchema, heightSchema } from './schemas';
+import { Event, Counter, Height } from './types';
+import { DB_KEY_ADDRESS_INFO, DB_LAST_COUNTER_INFO, DB_LAST_HEIGHT_INFO } from './constants';
+
+const { Database } = liskDB;
+type KVStore = liskDB.Database;
+
+// Returns DB's instance.
+export const getDBInstance = async (
+    dataPath: string,
+    dbName = 'lisk-framework-helloInfo-plugin.db',
+): Promise<KVStore> => {
+    const dirPath = join(dataPath.replace('~', os.homedir()), 'plugins/data', dbName);
+    await ensureDir(dirPath);
+    return new Database(dirPath);
+};
+
+// Returns event's data stored in the database.
+export const getEventHelloInfo = async (db: KVStore): Promise<(Event & { id: Buffer })[]> => {
+    // 1. Look for all the given key-value pairs in the database
+    const stream = db.createReadStream({
+        gte: Buffer.concat([DB_KEY_ADDRESS_INFO, Buffer.alloc(4, 0)]),
+        lte: Buffer.concat([DB_KEY_ADDRESS_INFO, Buffer.alloc(4, 255)]),
+    });
+    // 2. Get event's data out of the collected stream and push it in an array.    
+    const results = await new Promise<(Event & { id: Buffer })[]>((resolve, reject) => {
+        const ids: (Event & { id: Buffer })[] = [];
+        stream
+            .on('data', ({ key, value }: { key: Buffer; value: Buffer }) => {
+                ids.push({ ...codec.decode<Event>(offChainEventSchema, value), id: key.slice(DB_KEY_ADDRESS_INFO.length) });
+            })
+            .on('error', error => {
+                reject(error);
+            })
+            .on('end', () => {
+                resolve(ids);
+            });
+    });
+    return results;
+};
+
+// Stores event data in the database.
+export const setEventHelloInfo = async (db: KVStore, _lskAddress: Buffer, _message: string, _eventHeight: number, lastCounter: number): Promise<void> => {
+    const encodedAddressInfo = codec.encode(offChainEventSchema, { senderAddress: _lskAddress, message: _message, height: _eventHeight });
+    let dbKey = cryptography.utils.intToBuffer(lastCounter, 4);
```

Comment:
```suggestion
    const dbKey = Buffer.concat([DB_KEY_ADDRESS_INFO, cryptography.utils.intToBuffer(lastCounter, 4)]);
```
or `dbKey` should be renamed to something else